escrow/payment_channel_api.go

```diff
@@ -124,6 +124,9 @@ type PaymentChannelService interface {
 
 	//Get Channel from BlockChain
 	PaymentChannelFromBlockChain(key *PaymentChannelKey) (channel *PaymentChannelData, ok bool, err error)
+
+	// verifies whether storage channel nonce is equal to blockchain nonce or not
+	StorageNonceMatchesWithBlockchainNonce(key *PaymentChannelKey) (equal bool, err error)
```

Comment:
I would not add such specific function into `PaymentChannelService` interface. This function can be implemented in `state_service.go` module instead using 'PaymentChannelService.PaymentChannel()` and `PaymentChannelService.PaymentChannelFromBlockChain()' methods.

Reply:
fixed this

Reply:
@vsbogd , found a minor bug here .. fixing this